contracts/Rivendale.sol

```diff
@@ -132,9 +153,19 @@ contract Rivendale {
                 voteBreakdown[_id].walkerTally++;
             }
         }
-        if (voteBreakdown[_id].walkerCount > 0){
-            weightedVotes[0] = weights.walkerWeight * (voteBreakdown[_id].walkerTally / voteBreakdown[_id].walkerCount);
+
+        // Since walkers are a priviledged group, shouldn't the tally be based on the total amount of walkers? If there's huge abstention, a single walker can have 40% of voting power.

```

Comment:
It's about limiting groups.  So you would think that walkers would be a united group, customers a united group, and trb holders are an honest outside group....checks and balances